barretenberg/cpp/src/barretenberg/join_split_example/proofs/notes/circuit/value/witness_data.hpp

```diff
@@ -22,6 +18,7 @@ struct witness_data {
     witness_data(Builder& builder, native::value::value_note const& note)
     {
         secret = witness_ct(&builder, note.secret);
+        owner = group_ct(witness_ct(&builder, note.owner.x), witness_ct(&builder, note.owner.y), false);
         owner.x = witness_ct(&builder, note.owner.x);
         owner.y = witness_ct(&builder, note.owner.y);
```

Comment:
nit: we are doing `witness_ct(&builder, note.owner.x)` and `witness_ct(&builder, note.owner.y)` -- since its in join split which is only here in spirit, we don't need to change it